front-end/app/checkout/checkout.component.html

```diff
@@ -29,7 +29,7 @@
             </div>
 
             <h3>Order notes</h3>
-            <textarea class="form-control" rows="3" name="order_notes" [(ngModel)]="basket.orderNotes"></textarea>
+            <textarea class="form-control" rows="3" name="order_notes" [(ngModel)]="basket.orderNotes" placeholder="i.e. any adjustments to items or a arrange delivery for later..."></textarea>
```

Comment:
Feel free to merge once you've corrected the message and have double checked the DIFF to make sure you're merging the correct thing.